homeassistant/components/zwave_js/humidifier.py

```diff
@@ -32,6 +32,8 @@
 from .discovery import ZwaveDiscoveryInfo
 from .entity import ZWaveBaseEntity
 
+PARALLEL_UPDATES = 0
```

Comment:
I thought this was already the default for any async integration.

Reply:
Yep, looks like it https://github.com/home-assistant/core/blob/bf77c000eaf4b3e8ae6e607481f6a5a75cad9376/homeassistant/helpers/entity_platform.py#L141

Reply:
right and to be more explicit, we don't rely on polling at all, it's all push, so there are no platforms with an update method, sync or async

Reply:
Aaah yeah, ok that default is not very good. Maybe we also set it to 0 if we don't find any update method ? (So set it to 1 if we find `"update"` as a method.)

Reply:
would that still be potentially a concern if the services are synchronous? Is that even possible?

Reply:
Will merge this and think about it some more.